Source/WebCore/layout/integration/grid/LayoutIntegrationGridLayout.cpp

```diff
@@ -45,6 +46,13 @@ GridLayout::GridLayout(RenderGrid& renderGrid)
 {
 }
 
+void GridLayout::updateFormattingContextGeometries()
+{
+    auto boxGeometryUpdater = BoxGeometryUpdater { layoutState(), gridBox() };
+    boxGeometryUpdater.setFormattingContextRootGeometry(CheckedRef { gridBoxRenderer() }->containingBlock()->contentBoxLogicalWidth());
+    boxGeometryUpdater.setFormattingContextContentGeometry(CheckedRef { layoutState() }->geometryForBox(gridBox()).contentBoxWidth(), { });
```

Comment:
You should introduce `checkedGridBoxRenderer` and `checkedLayoutState` instead of wrapping in `CheckedRef`

Reply:
is this a repeating pattern?

Reply:
Not entirely sure how much, if any, it will continue to repeat. Happy to add it if we continue to use it! Also FWIW I see both usages throughout the codebase and it's not really clear (at least to me) when one should use one vs the other

Reply:
Looks like I'll need it at least one more time so I'll add it before I land the patch